README.md

```diff
@@ -5,7 +5,7 @@ SonarQube License-Check
 
 [![Maintainability](https://api.codeclimate.com/v1/badges/6ac787bb79b43e39c367/maintainability)](https://codeclimate.com/github/porscheinformatik/sonarqube-licensecheck/maintainability)
 
-This [SonarQube](http://www.sonarqube.org/) plugin ensures that projects use dependencies with compliant licenses. All dependencies and licenses can be viewed per projects and exported to CSV. This enables a simple governance of dependencies and licenses for the whole organization.
+This [SonarQube](http://www.sonarqube.org/) plugin ensures that projects use dependencies with compliant licenses. All dependencies and licenses can be viewed per projects and exported to Excel XLS. This enables a simple governance of dependencies and licenses for the whole organization.
```

Comment:
Actually the format is "Excel 2003 XML Format" (https://en.wikipedia.org/wiki/Microsoft_Office_XML_formats)

Reply:
ok